pdr_backend/pdr_dashboard/callbacks/callbacks_common.py

```diff
@@ -21,6 +26,28 @@ def display_page(pathname):
         result = get_page(pathname), get_navbar(NAV_ITEMS)
         return result
 
+    @app.callback(
+        Output("start-date", "data"),
+        [Input("general-lake-date-period-radio-items", "value")],
+    )
+    def set_period_start_date(selected_period_start):
+        start_date = (
+            get_start_date_from_period(int(selected_period_start))
+            if int(selected_period_start) > 0
+            else None
+        )
+        app.start_date = UnixTimeMs(start_date * 1000) if start_date else None
```

Comment:
please use the utils from time classes, no multiplication or division with 1000 outside